### DISCUSSION TITLE
v4 beta

### DISCUSSION THREAD

**Discussion by RussKie** [gitextensions/gitextensions#10214] [General]
### Release Notes Highlights

* The app now requires .NET 6. The .NET runtime, if missing, will be installed when the application is executed.
* Improved repository load time considerably (#9243, #9735, #9864, etc) as well as other performance tweaks.
* Use native WSL Git executables for `\\wsl$` shares to speedup handling (#9702)
* Use the standard `FormBrowse` for file history and blame (#9445)
* BASE diff: Icons for unique changes (#9720)

### The following features and options have been removed:
* PuTTY binaries are no longer bundled with Git Extentions, please install PuTTY separately and change the path in settings.
* Theming is not supported - v3.x relied on EasyHook library to facilitate hooks into Windows API necessary for theming. However, .NET 6 does not support remoting and multiple AppDomain required for those hooks. To date no viable alternative has been found. https://github.com/gitextensions/gitextensions/pull/9191
* Removed TFS integration (https://github.com/gitextensions/gitextensions/pull/8522/commits/8190d0c0570af95ffbf9e168072d1b169eb99618)
* Removed JIRA integration (see: https://github.com/gitextensions/gitextensions/issues/9659)

![Downloads](https://img.shields.io/github/downloads/gitextensions/gitextensions/v4-beta/total.svg)

<hr /><em>This discussion was created from the release <a href='https://github.com/gitextensions/gitextensions/releases/tag/v4-beta'>v4 beta</a>.</em>

**Comment by chirontt** [comment]
Is there any plan to provide GE distribution compiled for Windows on Arm64?

**Comment by RussKie** [comment]
Eventually,  yes. This will require compiling as Any CPU and not as x86.
There were few attempts with mixed results.

**Reply by chirontt** [reply]
Is there a discussion thread in compiling for Windows on Arm64?

I'm attempting to compile GE for WoA, and would like to hear other people's experience & comments.

**Reply by gerhardol** [reply]
Only mentions so far. Open a new discussion.

**Reply by RussKie** [reply]
There shouldn't be any issues except the ConEmu not working, https://github.com/gitextensions/gitextensions/issues/9755#issuecomment-978683279.

**Reply by chirontt** [reply]
New discussion created. #10288
